src/Core/Kernel.php

```diff
@@ -350,42 +350,12 @@ protected function getCacheHash(): string
         ]);
     }
 
+    /**
+     * @deprecated tag:v6.8.0 - reason:remove - all connection variables are done in MySQLFactory
+     */
     protected function initializeDatabaseConnectionVariables(): void
     {
-        /**
-         * @deprecated tag:v6.7.0 - remove if-clause, we have already SQL_SET_DEFAULT_SESSION_VARIABLES which is documented does the same
-         */
-        $shopwareSkipConnectionVariables = EnvironmentHelper::getVariable('SHOPWARE_SKIP_CONNECTION_VARIABLES', false);
-
-        if ($shopwareSkipConnectionVariables) {
-            return;
-        }
-
-        $connection = self::getConnection();
-
-        try {
-            $setSessionVariables = (bool) EnvironmentHelper::getVariable('SQL_SET_DEFAULT_SESSION_VARIABLES', true);
-            $connectionVariables = [];
-
-            /**
-             * @deprecated tag:v6.7.0 - remove if clause and enforce timezone setting
-             */
-            $timeZoneSupportEnabled = (bool) EnvironmentHelper::getVariable('SHOPWARE_DBAL_TIMEZONE_SUPPORT_ENABLED', Feature::isActive('v6.7.0.0'));
```

Comment:
Do you want to enable this also in DAL by default now?  that env is also checked in services xml

Reply:
https://github.com/shopware/shopware/pull/6730/files#diff-9854b9bdce67aeecbbc86c20d848186142bf30968df539bf8885adf044b13a95R8

Is this enough?